Cargo.toml

```diff
@@ -7,3 +7,7 @@ members = [
 ]
 exclude = ["admin/rustfmt"]
 resolver = "2"
+
+[patch.crates-io]
+# TODO(XXX): Remove this once rcgen has cut a release w/ CRL support included. Only used in examples.
+rcgen = { git = 'https://github.com/est31/rcgen.git' }
```

Comment:
It might be worth pinging the upstream to cut a release to avoid this.

I've been holding off on that only because I'm hoping to add support for the cert CRL distribution point and CRL issuing distribution point extensions first. 

I might be able to find time to work on that ~today, but perhaps it shouldn't block a release either way?

Reply:
> I might be able to find time to work on that ~today, but perhaps it shouldn't block a release either way?

Depends on whether those changes would need to be semver-breaking.

Reply:
I think it can be done w/o breaking semver for the cert CRL distrib point because the `CertificateParams` struct is non-exhaustive.

For the CRL issuing distribution point I think it will need to be breaking, I didn't mark the `CertificateRevocationListParams` struct as non-exhaustive (in retrospect, an oversight).